teps/0001-tekton-enhancement-proposal-process.md

```diff
@@ -0,0 +1,471 @@
+
+For example, the TEP template used to track API changes will
+likely have different subsections than the template for proposing
+governance changes. However, as changes start impacting other WGs or
+the larger developer communities outside of a WG, the TEP process
+should be used to coordinate and communicate.
+
+
+### TEP Metadata
+
+There is a place in each TEP for a YAML document that has standard
+metadata. This will be used to support tooling around filtering and
+display. It is also critical to clearly communicate the status of a
+TEP.
+
+Metadata items:
+* **title** Required
+  * The title of the TEP in plain language. The title will also be
+    used in the TEP filename. See the template for instructions and
+    details.
+* **status** Required
+  * The current state of the TEP.
+  * Must be one of `proposed`, `implementable`,
+    `implemented`,`withdrawn`, or `replaced`.
+* **authors** Required
+  * A list of authors for the TEP. This is simply the github ID. In
+    the future we may enhance this to include other types of
+    identification.
+* **creation-date** Required
+  * The date that the TEP was first submitted in a PR.
+  * In the form `yyyy-mm-dd`
+  * While this info will also be in source control, it is helpful to
+    have the set of TEP files stand on their own.
+* **last-updated** Optional
+  * The date that the TEP was last changed significantly.
+  * In the form `yyyy-mm-dd`
+* **see-also** Optional
+  * A list of other TEPs that are relevant to this TEP.
+  * In the form `TEP-123`
+* **replaces** Optional
+  * A list of TEPs that this TEP replaces. Those TEPs should list
+    this TEP in their `superseded-by`.
+  * In the form `TEP-123`
+* **superseded-by** Optional
+  * A list of TEPs that supersede this TEP. Use of this should be
+    paired with this TEP moving into the `Replaced` status.
+  * In the form `TEP-123`
+
+
+### TEP Workflow
+
+A TEP has the following states
+
+- `proposed`: The TEP has been proposed and is actively being
+  defined. This is the starting state while the TEP is being fleshed
+  out and actively defined and discussed.
+- `implementable`: The approvers have approved this TEP for
+  implementation.
+- `implemented`: The TEP has been implemented and is no longer
+  actively changed. From that point on, the TEP should be considered
+  *read-only*.
+- `withdrawn`: The TEP has been withdrawn by the authors or by the
+  community on agreement with the authors.
+- `replaced`: The TEP has been replaced by a new TEP. The
+  `superseded-by` metadata value should point to the new TEP.
+
+When a TEP is merged with the `proposed` state, this means the
+project owners acknowledge this is something we need to work on *but*
+the proposal needs to be more detailed before we can go ahead and
+implement it in the main project(s). This state doesn't prevent using
+`tektoncd/experimental` to *experiment* and gather feedback.
+
+A TEP can be created with the `implementable` state if it doesn't need
```

Comment:
NIT: Maybe reword to something like: A TEP can be moved to the `implementable` state if it doesn't need any more discussion and is approved as is.